Automattic/wp-calypso
# Fixed text issues and image loading issues

#### Changes proposed in this Pull Request

* The initial image grid loading for the landing page had been slow. This was fixed by including a single image to be loaded.

| Before      | After |
| ----------- | ----------- |
| ![licecap](https://user-images.githubusercontent.com/3422709/133212191-7fecc2b1-60d0-4e14-9d38-444e1fd47f1f.gif)  |  ![licecap-new](https://user-images.githubusercontent.com/3422709/133212236-035fcc2a-4320-48c0-bf0d-412cbdd45f54.gif)    |

* Fixes to text suggested in the last call

#### Testing instructions
* Load the landing page and the thank you page and check that the lead time is changed to 3-4 business days
* The image should load once rather than in a grid


Related to #
